Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2489,3 +2489,17 @@ extension BrowserViewController {
         self.present(alert, animated: true, completion: nil)
     }
 }
+
+extension BrowserViewController {
+    static func foregroundBVC() -> BrowserViewController {
+//        if #available(iOS 13.0, *) {
```

Comment:
This won't compile yet, but it will be needed for multiwindow support, so it is commented out.